### ISSUE TITLE
[FEATURE] [0.9.2] Concerns about locking NPC Activations and max Structure/Stress 

### ISSUE THREAD

**Issue by VexWerewolf** [Eranziel/foundryvtt-lancer#194]
**Is your feature request related to a problem? Please describe.**
![Max Structure Stress](https://user-images.githubusercontent.com/83786280/117662875-f562c100-b197-11eb-8597-afe19b2a5e5a.jpg)

Currently, the Activations and maximum Structure/Stress of an NPC are not directly editable. One of my players, who's also a GM, raised concerns about this, due to the fact that it doesn't account for things like Ultras getting three activations if there's five or more players. Maximum Structure/Stress is slightly less of an issue, since they are mostly determined by template features, but there are probably edge cases in which a GM would want to directly edit the maximum Structure/Stress of an NPC without recourse to traits.

**Describe the solution you'd like**
I'm really happy that 0.9.2 automates the process of setting Activations and max Structure/Stress automatically according to traits, but I think GMs should still be able to edit the maximum values manually if they want to.

**Discord contact info**
Vex#1075

**Comment by ScyllPoesis** [comment]
This is currently doable but not in the way you're thinking. The correct way to do this is in the new data structure (I could be corrected by white) is to edit the providing template or class itself, as the stats and derived data is just an assembly of the base information the mech owns, like legos.

First off, `activations` are currently just a reference and not automatically done due to some patch work for Lancer-Initiative. Currently you add more or less activations on the combat tracker itself, which will be getting some work done for 1.0 or shortly after. But assuming that gets functional again, you have the above two things to mess with to mess with activations, structure/stress, or anything really: Templates/Traits and Classes.  
1. Templates are just collections of traits that get auto-loaded onto the mech when applied, so this is actually talking about traits. For Ultras the trait is `SHOCK AND AWE` and it gives conditionally two or three activations depending on the number of hostiles. For ease we set it to 2 and if needed the GM can manually give them another on the tracker. At the data level this item has a bonus of `{activations: 1}`, which propagates up once the mech data is assembled. Unfortunately our Trait sheets are not updated yet, so editing these bonuses and other homebrew trait support isn't fully there yet. This brings us to the arguably easier but less clear route.
2. Classes (once dragged to an NPC) are individual items, and so can be edited directly. Classes form the base data for all NPCs and so represent the core stats that you edit. The only problem is that once you start editing the class stats it isn't clear what they were to begin with, but hey you're the GM.

tl;dr Stats are all still readily available for modification, just not on the sheet itself. Click the class (in this case the `Mirage` item) on the NPC's sheet and you can increment/decrement all the fields without affecting other NPCs of that class.

**Comment by ScyllPoesis** [comment]
Yep, the above also works as bonuses are additive, as long as you're fine with doubled up traits.

**Comment by Grygon** [comment]
Closing this since these values aren't actually editable in any way on the NPC--you'd have to go to the source item to edit
